src/test/mockData.ts

```diff
@@ -98,8 +98,8 @@ export const mockData: MockCity =
             food: {
                 color: "green",
                     ideas: [{
-                            name: "idea name",
-                            description: "lorem ipsum dolor sit amet ...",
+                            name: "Guachinche Ivan",
+                            description: "Go to Guachinche Ivan in La Laguna and take a rabbit",
```

Comment:
🤪 I think that last coma shouldn't be there.

Reply:
that last comma don do anything. Is a good practice to add it in order to don 't forget them between lines